### ISSUE TITLE
Find Measures doesn't show all BCL Measures

### ISSUE THREAD

**Issue by MatthewSteen** [openstudiocoalition/OpenStudioApplication#178]
Issue overview
--------------
With a fresh install of OSA 1.0 on Ubuntu 18.04 the user is not able to find or download all measures from the BCL. Appears to be limited to `Whole Building > Space Types`. 

Workaround is to use the search feature.

## Current Behavior
Whole Building > Space Types does not show _any_ measures.

![Screenshot from 2020-05-31 19-33-36](https://user-images.githubusercontent.com/8397034/83368337-ab614080-a375-11ea-94f7-fe639027cbca.png)

## Expected Behavior
Whole Building > Space Types should have 23 measures available according to the BCL.

## Steps to Reproduce

1. Open the app
2. Components & Measures > Find Measures (or Measures tab > Find Measures on BCL)
3. Whole Building > Space Types
4. it's empty

### Environment

Some additional details about your environment for this issue (if relevant):
 - Platform (Operating system, version): Ubuntu 18.04
 - Version of OpenStudioApplication (if using an intermediate build, include SHA): 1.0

### Context

Trying to download the `Create DOE Prototype Building` measure.

**Comment by DavidGoldwasser** [comment]
I can confirm this on mac. But it only happens on certain categories or sub-categories. for example I can see content on Whole Building / Whole Building Schedules, but not Whole Building / Space Types or the parent "Whole Building". "Electric Lighting and its sub-categories appear to work. I also confirmed that the search still works fine with the 2.9.1 application.

@MatthewSteen what search term did you use? I found on mac that if I use a space in the search term I get a crash with this dialog. I confirmed the crash does not happen with 2.9.1
<img width="433" alt="Screen Shot 2020-06-09 at 8 18 46 AM" src="https://user-images.githubusercontent.com/4857055/84159326-30112600-aa2a-11ea-8103-1834302738c7.png">
